websites/R/R-a-d.io/presence.ts

```diff
@@ -0,0 +1,91 @@
+const presence = new Presence({
+		//The client ID of the Application created at https://discordapp.com/developers/applications
+		clientId: "1073136952905318401",
+	}),
+	// Get current player status
+	playStatus = (): string => {
+		let status: string;
+		const playButtonText: string =
+			document.querySelector("#stream-play").textContent;
+		if (playButtonText === "Play Stream") status = "Stopped";
+		else if (playButtonText === "Stop Stream") status = "Playing";
+		else status = playButtonText;
+		return status;
+	},
+	// Get current song playing
+	currentSong = (): string => {
+		return document.querySelector("#np").textContent;
+	};
+
+presence.on("UpdateData", async () => {
+	// Current path
+	const path = window.location.pathname,
```

Comment:
`document.location` is preferred over `window.location` in this repository.
```suggestion
	const path = document.location.pathname,
```